pkg/syncer/metadatasyncer.go

```diff
@@ -302,10 +302,20 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 		IsWorkloadDomainIsolationSupported = commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx,
 			common.WorkloadDomainIsolation)
 		if !IsWorkloadDomainIsolationSupported {
-			go k8sorchestrator.HandleLateEnablementOfCapability(ctx, clusterFlavor, common.WorkloadDomainIsolation,
+			go commonco.ContainerOrchestratorUtility.HandleLateEnablementOfCapability(ctx, clusterFlavor,
+				common.WorkloadDomainIsolation,
 				"", "")
-		}
-		if IsWorkloadDomainIsolationSupported {
+		} else {
+			IsWorkloadDomainIsolationFSSEnabled :=
+				commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.WorkloadDomainIsolationFSS)
+			if !IsWorkloadDomainIsolationFSSEnabled {
+				err = commonco.ContainerOrchestratorUtility.EnableFSS(ctx, common.WorkloadDomainIsolationFSS)
```

Comment:
Can you please add comments at few places that we are still enabling internal FSS for WLDI on supervisor to support cases where Guest is still old and supervisor is upgraded.

Reply:
added comments to explain why we are enabling FSS in the configmap